map_model/src/objects/traffic_signals.rs

```diff
@@ -256,6 +299,20 @@ impl Stage {
             }
         }
     }
+    pub fn check_minimum_crosswalk_time(&mut self, movement: &Movement) {
```

Comment:
"check" sounds like it'll return a bool or print a warning, but this actually fixes things up. Let's name this `enforce_minimum_crosswalk_time` or `fix_minimum_crosswalk_time`

Reply:
Yeah that's a good point. Fixed